packages/core/src/hydration/event_delegation.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ENVIRONMENT_INITIALIZER, Injector} from '../di';
+import {inject} from '../di/injector_compatibility';
+import {Provider} from '../di/interface/provider';
+import {setStashFn} from '../render3/instructions/listener';
+import {
+  GlobalEventDelegation,
+  initGlobalEventDelegation,
+  sharedStashFunction,
+} from '../event_delegation_utils';
+
+import {IS_GLOBAL_EVENT_DELEGATION_ENABLED} from './tokens';
+
+/**
+ * Returns a set of providers required to setup support for event delegation.
+ */
+export function withGlobalEventDelegation(): Provider[] {
```

Comment:
We can use a common convention here and start with `provide*`:
```suggestion
export function provideGlobalEventDelegation(): Provider[] {
```

Reply:
done